wrangler-core/src/main/java/io/cdap/directives/validation/ValidateStandard.java

```diff
@@ -0,0 +1,213 @@
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+package io.cdap.directives.validation;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+import io.cdap.cdap.api.annotation.Description;
+import io.cdap.cdap.api.annotation.Name;
+import io.cdap.cdap.api.annotation.Plugin;
+import io.cdap.directives.validation.Manifest.Standard;
+import io.cdap.directives.validation.conformers.Conformer;
+import io.cdap.directives.validation.conformers.JsonConformer;
+import io.cdap.wrangler.api.Arguments;
+import io.cdap.wrangler.api.Directive;
+import io.cdap.wrangler.api.DirectiveExecutionException;
+import io.cdap.wrangler.api.DirectiveParseException;
+import io.cdap.wrangler.api.ErrorRowException;
+import io.cdap.wrangler.api.ExecutorContext;
+import io.cdap.wrangler.api.ReportErrorAndProceed;
+import io.cdap.wrangler.api.Row;
+import io.cdap.wrangler.api.annotations.Categories;
+import io.cdap.wrangler.api.parser.ColumnName;
+import io.cdap.wrangler.api.parser.Identifier;
+import io.cdap.wrangler.api.parser.TokenType;
+import io.cdap.wrangler.api.parser.UsageDefinition;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+
+/**
+ * A directive for validating data against a number of built-in data models.
+ */
+@Plugin(type = Directive.TYPE)
+@Name(ValidateStandard.NAME)
+@Categories(categories = {"data-quality"})
+@Description("Checks a column against a standard schema")
+public class ValidateStandard implements Directive {
+
+  public static final String NAME = "validate-standard";
+
+  private static final Logger LOG = LoggerFactory.getLogger(ValidateStandard.class);
+
+  private static final String STANDARD_SPEC = "standard-spec";
+  private static final String COLUMN = "column";
+
+  private static final Map<String, Conformer<JsonObject>> schemaToConformer = new HashMap<>();
+  private static Manifest standardsManifest = null;
+
+  static final String SCHEMAS_RESOURCE_PATH = "schemas/";
+  static final String MANIFEST_PATH = SCHEMAS_RESOURCE_PATH + "manifest.json";
+
+  static final Map<String, Conformer.Factory<JsonObject>> FORMAT_TO_FACTORY = new HashMap<>();
+
+  private String column;
+  private String schema;
+
+  static {
+    FORMAT_TO_FACTORY.put(JsonConformer.SCHEMA_FORMAT, new JsonConformer.Factory());
+
+    try {
+      standardsManifest = getManifest();
+    } catch (IOException e) {
+      LOG.error("Unable to read standards manifest", e);
+    }
+  }
+
+  @Override
+  public UsageDefinition define() {
+    UsageDefinition.Builder builder = UsageDefinition.builder(NAME);
+    builder.define(COLUMN, TokenType.COLUMN_NAME);
+    builder.define(
+        STANDARD_SPEC,
```

Comment:
nit: indentation should be 2 spaces (applies to rest of PR)

Reply:
Done.